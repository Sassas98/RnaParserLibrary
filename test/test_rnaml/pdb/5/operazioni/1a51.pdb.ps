%!PS-Adobe-2.0
%%Title: A postscript generated from a RNAML file.
%%CreationDate: Sat May 11 22:22:01 2024
%%Orientation: Portrait
%%BoundingBox:    253   101   359   691

/Times-Bold findfont 12  scalefont setfont


/Times-Bold findfont 13  scalefont setfont    %%set font for (A U G C T)

%% stacking diagram widths for base-pairs 1 & 2
/W1 {1 setlinewidth} bind def
/W2 {1.5 setlinewidth} bind def
/W3 {2 setlinewidth} bind def
/W4 {3 setlinewidth} bind def

%% minor and major grooves filling color saturation
/MINOR_SAT 0.9 def
/MAJOR_SAT 0.1 def
/OTHER_SIDES {0 0 1 sethsbcolor fill} bind def

%% defineing geometry shapes
/NP {newpath} bind def
/CIRCLE {0 360 arc closepath} bind def                        %% circle 
/TRIANGLE {moveto lineto lineto closepath} bind def           %% triangle
/SQUARE {moveto lineto lineto lineto closepath} bind def      %% square
/LINE {moveto lineto stroke} bind def                         %% line
/DASHLINE { moveto lineto [2 4] 0 setdash stroke } bind def %% dashline
/DOTLINE  {1 0 1 setrgbcolor} bind def                        %% dotline
/Dw {1 setlinewidth} bind def
/FB {setgray fill} bind def
/R6 {moveto lineto lineto lineto lineto lineto closepath} bind def
/R9 {moveto lineto lineto lineto lineto lineto
     lineto lineto lineto closepath} bind def

%% line drawing colors for ACGITU & others
/Al {0.0000 1.00 1.00 sethsbcolor} bind def
/Cl {0.1667 1.00 1.00 sethsbcolor} bind def
/Gl {0.3333 1.00 1.00 sethsbcolor} bind def
/Il {0.3333 1.00 0.57 sethsbcolor} bind def
/Tl {0.6667 1.00 1.00 sethsbcolor} bind def
/Ul {0.5000 1.00 1.00 sethsbcolor} bind def
/Xl {0.0000 0.00 0.00 sethsbcolor} bind def
/XX {0.2000 0.20 0.20 sethsbcolor} bind def
/FILLBLACK {0.0000 0.00 0.00 sethsbcolor fill} bind def

%% minor groove filling colors for ACGITU & others
/Am {0.0000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Cm {0.1667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Gm {0.3333 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Im {0.3333 MINOR_SAT 0.57      sethsbcolor fill} bind def
/Tm {0.6667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Um {0.5000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Xm {0.0000 0.00      MAJOR_SAT sethsbcolor fill} bind def

%% major groove filling colors for ACGITU & others
/AM {0.0000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/CM {0.1667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/GM {0.3333 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/IM {0.3333 MAJOR_SAT 0.57      sethsbcolor fill} bind def
/TM {0.6667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/UM {0.5000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/XM {0.0000 0.00      MINOR_SAT sethsbcolor fill} bind def

%% define line width, line join style & cap style (1 means round)
1 setlinewidth 1 setlinejoin 1 setlinecap
   273   121 translate

/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
66.48 546.00  moveto (G) center show 
66.48 518.30  moveto (G) center show 
66.48 490.60  moveto (C) center show 
66.48 462.90  moveto (C) center show 
66.48 435.20  moveto (G) center show 
66.48 407.50  moveto (A) center show 
66.48 379.80  moveto (U) center show 
66.48 352.10  moveto (G) center show 
66.48 324.40  moveto (G) center show 
66.48 296.70  moveto (U) center show 
66.48 269.00  moveto (A) center show 
66.48 241.31  moveto (G) center show 
66.48 213.61  moveto (U) center show 
66.48 185.91  moveto (G) center show 
66.48 158.21  moveto (U) center show 
66.48 130.51  moveto (G) center show 
66.48 102.81  moveto (G) center show 
66.48 75.11  moveto (G) center show 
66.48 47.41  moveto (G) center show 
63.85 12.66  moveto (U) center show 
33.24 -4.00  moveto (C) center show 
2.63 12.66  moveto (U) center show 
0.00 47.41  moveto (C) center show 
0.00 75.11  moveto (C) center show 
0.00 102.81  moveto (C) center show 
0.00 130.51  moveto (C) center show 
0.00 158.21  moveto (A) center show 
0.00 185.91  moveto (U) center show 
0.00 213.61  moveto (G) center show 
0.00 241.31  moveto (C) center show 
0.00 269.00  moveto (G) center show 
0.00 296.70  moveto (A) center show 
0.00 324.40  moveto (G) center show 
0.00 352.10  moveto (A) center show 
0.00 379.80  moveto (G) center show 
0.00 407.50  moveto (U) center show 
0.00 435.20  moveto (A) center show 
0.00 462.90  moveto (G) center show 
0.00 490.60  moveto (G) center show 
0.00 518.30  moveto (C) center show 
0.00 546.00  moveto (C) center show 
0.005 setlinewidth 1 setlinejoin 1 setlinecap
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 66.48 556.00 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 556.00 moveto (3') center show grestore
Il 66.48 550.00 66.48 522.30 LINE
 Il 66.48 522.30 66.48 494.60 LINE
 Il 66.48 494.60 66.48 466.90 LINE
 Il 66.48 466.90 66.48 439.20 LINE
 Il 66.48 439.20 66.48 411.50 LINE
 Il 66.48 411.50 66.48 383.80 LINE
 Il 66.48 383.80 66.48 356.10 LINE
 Il 66.48 356.10 66.48 328.40 LINE
 Il 66.48 328.40 66.48 300.70 LINE
 Il 66.48 300.70 66.48 273.00 LINE
 Il 66.48 273.00 66.48 245.31 LINE
 Il 66.48 245.31 66.48 217.61 LINE
 Il 66.48 217.61 66.48 189.91 LINE
 Il 66.48 189.91 66.48 162.21 LINE
 Il 66.48 162.21 66.48 134.51 LINE
 Il 66.48 134.51 66.48 106.81 LINE
 Il 66.48 106.81 66.48 79.11 LINE
 Il 66.48 79.11 66.48 51.41 LINE
 Il 66.48 51.41 63.85 16.66 LINE
 Il 63.85 16.66 33.24 0.00 LINE
 Il 33.24 0.00 2.63 16.66 LINE
 Il 2.63 16.66 0.00 51.41 LINE
 Il 0.00 51.41 0.00 79.11 LINE
 Il 0.00 79.11 0.00 106.81 LINE
 Il 0.00 106.81 0.00 134.51 LINE
 Il 0.00 134.51 0.00 162.21 LINE
 Il 0.00 162.21 0.00 189.91 LINE
 Il 0.00 189.91 0.00 217.61 LINE
 Il 0.00 217.61 0.00 245.31 LINE
 Il 0.00 245.31 0.00 273.00 LINE
 Il 0.00 273.00 0.00 300.70 LINE
 Il 0.00 300.70 0.00 328.40 LINE
 Il 0.00 328.40 0.00 356.10 LINE
 Il 0.00 356.10 0.00 383.80 LINE
 Il 0.00 383.80 0.00 411.50 LINE
 Il 0.00 411.50 0.00 439.20 LINE
 Il 0.00 439.20 0.00 466.90 LINE
 Il 0.00 466.90 0.00 494.60 LINE
 Il 0.00 494.60 0.00 522.30 LINE
 Il 0.00 522.30 0.00 550.00 LINE
 /Times findfont 8 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Tl -11.00 518.30 moveto (107) center show grestore
gsave Tl 75.48 435.20 moveto (72) center show grestore
gsave Tl -11.00 379.80 moveto (102) center show grestore
gsave Tl 75.48 296.70 moveto (77) center show grestore
gsave Tl -11.00 241.30 moveto (97) center show grestore
gsave Tl 77.48 158.21 moveto (82) center show grestore
gsave Tl -11.00 102.81 moveto (92) center show grestore
gsave Tl 73.92 6.14 moveto (87) center show grestore
NP W1
 Xl 58.48 548.00 8.00 548.00 LINE
 Xl 58.48 552.00 8.00 552.00 LINE
 NP W1
 Xl 58.48 520.30 8.00 520.30 LINE
 Xl 58.48 524.30 8.00 524.30 LINE
 NP W1
 Xl 58.48 492.60 8.00 492.60 LINE
 Xl 58.48 496.60 8.00 496.60 LINE
 NP W1
 Xl 58.48 464.90 8.00 464.90 LINE
 Xl 58.48 468.90 8.00 468.90 LINE
 NP W1
 Xl 58.48 439.20 41.24 439.20 LINE
 Xl 36.24 436.20 36.24 442.20 41.24 439.20 TRIANGLE
 gsave  grestore stroke
 Xl 36.24 439.20 30.24 439.20 LINE
 Xl 30.24 436.20 25.24 436.20 25.24 442.20 30.24 442.20 SQUARE
 gsave  grestore stroke
 Xl 25.24 439.20 8.00 439.20 LINE
 NP W1
 Xl 58.48 411.50 41.24 411.50 LINE
 Xl 41.24 408.50 36.24 408.50 36.24 414.50 41.24 414.50 SQUARE
 gsave  grestore stroke
 Xl 36.24 411.50 30.24 411.50 LINE
 Xl 27.74 411.50 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 25.24 411.50 8.00 411.50 LINE
 NP W1
 Xl 58.48 383.80 41.24 383.80 LINE
 Xl 41.24 380.80 36.24 380.80 36.24 386.80 41.24 386.80 SQUARE
 gsave  grestore stroke
 Xl 36.24 383.80 30.24 383.80 LINE
 Xl 27.74 383.80 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 25.24 383.80 8.00 383.80 LINE
 NP W1
 Xl 58.48 328.40 41.24 328.40 LINE
 Xl 38.74 328.40 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 36.24 328.40 30.24 328.40 LINE
 Xl 30.24 325.40 25.24 325.40 25.24 331.40 30.24 331.40 SQUARE
 gsave  grestore stroke
 Xl 25.24 328.40 8.00 328.40 LINE
 NP W1
 Xl 58.48 300.70 41.24 300.70 LINE
 Xl 38.74 300.70 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 36.24 300.70 30.24 300.70 LINE
 Xl 30.24 297.70 25.24 297.70 25.24 303.70 30.24 303.70 SQUARE
 gsave  grestore stroke
 Xl 25.24 300.70 8.00 300.70 LINE
 NP W1
 Xl 58.48 273.00 41.24 273.00 LINE
 Xl 41.24 270.00 36.24 270.00 36.24 276.00 41.24 276.00 SQUARE
 gsave  grestore stroke
 Xl 36.24 273.00 30.24 273.00 LINE
 Xl 30.24 270.00 30.24 276.00 25.24 273.00 TRIANGLE
 gsave  grestore stroke
 Xl 25.24 273.00 8.00 273.00 LINE
 NP W1
 Xl 58.48 243.31 8.00 243.30 LINE
 Xl 58.48 247.31 8.00 247.30 LINE
 NP W1
 Xl 33.24 217.61 3.00 CIRCLE 
 gsave  grestore stroke
 NP W1
 Xl 33.24 189.91 3.00 CIRCLE 
 gsave  grestore stroke
 NP W1
 Xl 58.48 162.21 8.00 162.21 LINE
 NP W1
 Xl 58.48 132.51 8.00 132.51 LINE
 Xl 58.48 136.51 8.00 136.51 LINE
 NP W1
 Xl 58.48 104.81 8.00 104.81 LINE
 Xl 58.48 108.81 8.00 108.81 LINE
 NP W1
 Xl 58.48 77.11 8.00 77.11 LINE
 Xl 58.48 81.11 8.00 81.11 LINE
 Al W2 60.48 356.10 6.00 356.10 gsave  DASHLINE grestore
 Al W2 60.48 51.41 6.00 51.41 gsave  DASHLINE grestore
 
 showpage 
