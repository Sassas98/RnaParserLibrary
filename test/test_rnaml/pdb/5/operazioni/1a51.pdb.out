PDB data file name: 1a51.pdb_new
-----------------------------------------------------------
BEGIN_base-pair
     1_41, A:    68 G-C   108 A: +/+ cis         XIX
     2_40, A:    69 G-C   107 A: +/+ cis         XIX
     3_39, A:    70 C-G   106 A: +/+ cis         XIX
     4_38, A:    71 C-G   105 A: +/+ cis         XIX
     5_37, A:    72 G-A   104 A: S/H tran        XI
     6_36, A:    73 A-U   103 A: H/W tran        XXIV
     7_35, A:    74 U-G   102 A: H/W tran        n/a
     9_33, A:    76 G-G   100 A: W/H tran        VII
    10_32, A:    77 U-A    99 A: W/H tran        XXIV
    11_31, A:    78 A-G    98 A: H/S tran        XI
    12_30, A:    79 G-C    97 A: +/+ cis         XIX
    13_29, A:    80 U-G    96 A: W/W cis         XXVIII
    14_28, A:    81 G-U    95 A: W/W cis         XXVIII
    15_27, A:    82 U-A    94 A: -/- cis         XX
    16_26, A:    83 G-C    93 A: +/+ cis         XIX
    17_25, A:    84 G-C    92 A: +/+ cis         XIX
    18_24, A:    85 G-C    91 A: +/+ cis         XIX
    26_27, A:    93 C-A    94 A:      stacked
    27_28, A:    94 A-U    95 A:      stacked
    29_30, A:    96 G-C    97 A:      stacked
    36_37, A:   103 U-A   104 A:      stacked
     8_34, A:    75 G-A   101 A: H/W cis         !1H(b_b).
    19_23, A:    86 G-C    90 A: W/W cis         !1H(b_b).
END_base-pair
  The total base pairs =  17 (from   41 bases)
------------------------------------------------
 Standard  WW--cis  WW-tran  HH--cis  HH-tran  SS--cis  SS-tran
        9        2        0        0        0        0        0
  WH--cis  WH-tran  WS--cis  WS-tran  HS--cis  HS-tran
        0        4        0        0        0        2
------------------------------------------------
